tidb-cloud/changefeed-overview.md

```diff
@@ -28,10 +25,20 @@ To create a changefeed, refer to the tutorials:
 - [Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md) (Beta)
 - [Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md)
 
-## Delete a changefeed
+## Query TiCDC RCUs
 
 1. Navigate to the cluster overview page of the target TiDB cluster, and then click **Changefeed** in the left navigation pane.
-2. Locate the corresponding changefeed you want to delete, and click **...** > **Delete** in the **Action** column.
+2. Locate the corresponding changefeed you want to query, and click **...** > **View** in the **Action** column.
+3. You can see the current TiCDC Replication Capacity Units (RCUs) in the **Specification** area of the page.
+
+## Scale a changefeed
+
+You can change the TiCDC Replication Capacity Units (RCUs) of a changefeed by scaling up or down the changfeed.
+
+1. Navigate to the cluster overview page of the target TiDB cluster, and then click **Changefeed** in the left navigation pane.
+2. Locate the corresponding changefeed you want to scale, and click **...** > **Scale Up/Down** in the **Action** column.
+3. Select a new specification.
+4. Click **Submit**.
 
```

Comment:
```suggestion
```suggestion
There are some limitations:
1. Changefeeds created before March 28th 2023 cannot be scaled up or down. Similarly, newly created changefeeds within the same TiDB cluster cannot be scaled up or down.
2. Only TiDB clusters without existing changefeeds support scaling up and down for newly created changefeeds.
```
```